### PR TITLE
Fine-tune LayoutLM to extract Ex. 21

### PR DESCRIPTION
**Pull request by katie-lamb**
Create a module for creating Label Studio inputs, handling and formatting outputs from Label Studio, and fine-tuning a LayoutLM model for Ex. 21 extraction.

I've added functions that roughly break down into the following categories:

**Label Studio Input Creation**

* Use `create_labeled_dataset.create_inputs_for_label_studio` to create a JSON and an image that can be uploaded to GCS and then synced with Label Studio. Currently this requires reading in PDF text from the SEC 10K local cache created by the metadata archiver. 

**Label Studio output formatting**

* Use `rename_labeled_filings` to rename the Label Studio outputs that are synced with GCS. The objects in the buckets aren't named the same as their corresponding inputs, so this script renames these objects to have a filename that matches up with its SEC filename. 
* Use `create_labeled_dataset.format_as_ner_annotations` to take output JSONs from Label Studio and format them as named entity recognition annotations that can then be used for fine-tuning LayoutLM. This currently depends on having the output JSONs from Label Studio downloaded locally, but it should be an easy fix to pull these from the "labeled" bucket in GCS instead.

**LayoutLM Fine-tuning**

* In `finetune_ex21_extractor.py`, pass the NER annotations generated from `create_labeled_dataset.format_as_ner_annotations` into `LayoutLMFineTuner.from_ner_annotations` to create an instance of LayoutLMFineTuner that is ready to train. Fine-tune it with `LayoutLMFineTuner.train_model()`. Inference and evaluation of the model currently lives in notebook 7 and hasn't been integrated into the model.

### CODE REVIEW

**Comment by zschira** [comment]
I think I've got a pretty good handle on what's going on here, so I'm going to go ahead and merge and start working on mlflow integration. There is a CI issue from missing GCS credentials, which I'll try to address as well.

**Comment by katie-lamb** [comment]
@zschira Btw I think this GCS credentials CI issue is from the `rename_labeled_filings` script that I made where I probably inappropriately am using the GCSArchiver class to access the GCS labeled filings bucket. I think I did this kind of lazily to get access to that bucket but it's probably not right to use that class.
